lib/models/github.dart

```diff
@@ -301,3 +301,16 @@ class GithubContentReferenceItem {
   factory GithubContentReferenceItem.fromJson(Map<String, dynamic> json) =>
       _$GithubContentReferenceItemFromJson(json);
 }
+
+@JsonSerializable(fieldRename: FieldRename.snake) 
+class GithubContributorItem {
+  int id;
+  String login;
+  String avatarUrl;
+  String htmlUrl;
+  int contributions;
+  GithubContributorItem();
+  factory GithubContributorItem.fromJson(Map<String, dynamic> json) =>
+    _$GithubContributorItemFromJson(json);
+}
+
```

Comment:
Seems `github.g.dart` changes are missing?

Reply:
Sorry, I thought since it is a generated file, I shouldn't include it. Will do.